index2.php

```diff
@@ -0,0 +1,18 @@
+<html lang="pl">
```

Comment:
Jesli skrypt tylko echuje to po co ten html?

Reply:
Poza tym, jesli chcesz pomieszac html z php to taki plik powinien miec rozszerzenie .phtml